[BUG] Type 'number' is not assignable to type 'string'.

Importing and setting icon prop like so `<IconSearch size={10} />` but on build i get error 


```
node_modules/.pnpm/@tabler+icons@1.84.0_biqbaboplfbrettd7655fr4n2y/node_modules/@tabler/icons/icons-react/index.d.ts:3:11 - error TS2430: Interface 'TablerIconProps' incorrectly extends interface 'SVGAttributes<SVGElement>'.
  Types of property 'stroke' are incompatible.
    Type 'string | number | undefined' is not assignable to type 'string | undefined'.
      Type 'number' is not assignable to type 'string'.

3 interface TablerIconProps extends SVGAttributes<SVGElement> { color?: string; size?: string | number; stroke?: string | number; }
            ~~~~~~~~~~~~~~~


Found 1 error in node_modules/.pnpm/@tabler+icons@1.84.0_biqbaboplfbrettd7655fr4n2y/node_modules/@tabler/icons/icons-react/index.d.ts:3
```

![Screen Shot 2022-08-20 at 19 53 06](https://user-images.githubusercontent.com/36937039/185747086-b6cef82d-a2a1-4db4-a537-12fc6b83bf66.jpg)

![Screen Shot 2022-08-20 at 19 55 32](https://user-images.githubusercontent.com/36937039/185747154-69e090b2-97fa-4615-965f-24495db9e9d3.jpg)

--- a/gulpfile.js
+++ b/gulpfile.js
@@ -662,7 +662,7 @@ gulp.task('svg-to-react', gulp.series('clean-react', async (cb) => {
   }
 
   let indexCode = '',
-      indexDCode = `import { FC, SVGAttributes } from 'react';\n\ninterface TablerIconProps extends SVGAttributes<SVGElement> { color?: string; size?: string | number; stroke?: string | number; }\n\ntype TablerIcon = FC<TablerIconProps>;\n\n`
+      indexDCode = `import { FC, SVGAttributes } from 'react';\n\ntype TablerIconProps = Omit<SVGAttributes<SVGElement>, 'color' | 'stroke'> & {\n  color?: SVGAttributes<SVGElement>['stroke'];\n  size?: SVGAttributes<SVGElement>['width'];\n  stroke?: SVGAttributes<SVGElement>['strokeWidth'];\n}\n\ntype TablerIcon = FC<TablerIconProps>;\n\n`
 
   await asyncForEach(files, async function(file) {
     const svgCode = optimizeSvgCode(fs.readFileSync(file).toString()),

--- a/icons-react/index.d.ts
+++ b/icons-react/index.d.ts
@@ -1,6 +1,10 @@
 import { FC, SVGAttributes } from 'react';
 
-interface TablerIconProps extends SVGAttributes<SVGElement> { color?: string; size?: string | number; stroke?: string | number; }
+type TablerIconProps = Omit<SVGAttributes<SVGElement>, 'color' | 'stroke'> & {
+  color?: SVGAttributes<SVGElement>['stroke'];
+  size?: SVGAttributes<SVGElement>['width'];
+  stroke?: SVGAttributes<SVGElement>['strokeWidth'];
+}
 
 type TablerIcon = FC<TablerIconProps>;
 

